### PR TITLE
add default_application to MimeTypeInfo

### PR DESCRIPTION
**Pull request by wkloucek**
needed by https://github.com/cs3org/reva/pull/2230

### CODE REVIEW

File: cs3/app/registry/v1beta1/resources.proto [hunk 1]
```diff
@@ -96,4 +96,7 @@ message MimeTypeInfo {
   // Defaults to false, i.e. files with this mime type can be opened
   // but not directly allow_creationd from the web UI.
```
Comment (glpatcern) [inline cs3/app/registry/v1beta1/resources.proto:97]:
```suggestion
  // but not directly created from the web UI.
```

**Review by glpatcern — APPROVED** [review]
Looks good to me, can you please also apply the (unrelated) suggested change?

**Comment by glpatcern** [comment]
Thanks Willy, I don't get why the build is failing though: could you perhaps try and `make` so all artifacts are regenerated?

**Comment by wkloucek** [comment]
> Thanks Willy, I don't get why the build is failing though: could you perhaps try and `make` so all artifacts are regenerated?

I already posted in the CS3apis Gitter channel about this. I think the changes fromhttps://github.com/cs3org/cs3apis-build/pull/18 broke it.

**Comment by wkloucek** [comment]
@glpatcern CI is green now

**Pull request merged**